113.md

```diff
@@ -0,0 +1,141 @@
+NIP-113
+===
+Activity Events
+----------------
+`draft` `optional`
+
+This NIP defines `kind:30100`: a parameterized replaceable event to describe activities. Activities are things that one might record via a tracking watch (garmin, fitbit, apple watch, etc...) and wish to share with other.
+
+In general, activities are notes that have an activity file associated with them (gpx, tcx, fit), and a summary can be produced about this activity.
+
+The structure of an activity follows very closely with [NIP-23](23.md) long-form content events.
+### Content
+The `.content` field MAY contain a description of the activity
+### Activity Types
+Activities MUST include a `t` tag with an activity type (defined in Appendix A). e.g. `run | walk | hike`
+### Activity File
+Activities MAY include an `r` tag, which will include a URL to the location where an activity file can be found.
+### Metadata
+`t` - The activity type  
+`r` - location of activity file (OPTIONAL)  
+`title` - The title of the activity  
+`recorded_at` - The unix timestamp (in seconds) when the activity was recorded  
+`encrypted` - An indication that this event is encrypted (OPTIONAL)
+
+```json
+{
+  "kind": 30100,
+  "content": "<description-of-activity>",
+  "tags": [
+	 ["title", "Title of Activity"],
+	 ["t", "<activity-type-from-nip>"]
+	 ["d", "UUID generated by client"],
+	 ["r", "url to download activity file"],
+	 ["published_at", "<unix-timestamp>"],
+	 ["updated_at", "<unix-timestamp>"],
+	 ["recorded_at", "<unix-timestamp>"],
+	 ["encrypted"] // Optional, activity file is encrypted
+  ],
+}
+```
+### Summaries
+`kind:30101` defines an event which is meant to summarize activities, users, places, etc...
+Summaries are intended to capture a wide range things that users may wish to summarize in relation to activities. 
+Users will likely want a summary of a specific event (e.g. milage, time, pace, etc...), a summary for a group of events (e.g. miles run this month), a summary of locations (e.g. heatmaps based on a geohash). 
+
+This NIP does not attempt to describe all the different summaries that users could want, as this list will be ever evolving. Rather this event type should facilitate clients to retrieve relevant summaries for a user.
+#### Summary Metadata
+`r` - An ID that this summary pertains to. It could be an activity ID or a user pubkey  
```

Comment:
In case of a user pubkey, a p-tag can be used.
In case of a activity ID, is this the ID of the 30100 event? If so, a a-tag can be used to reference to an event.

Reply:
if it's a replaceable event and you've since replaced it, wouldn't you still want a reference to the most up to date activity ID, rather than the eventID that might be stale?